src/services/MetaService.ts

```diff
@@ -68,13 +78,15 @@ export class MetaService {
    * Make http request to get meta data. Response data will be parsed, then the Promise will be resolved.
    */
   async get(requested: string[], layout?: string): Promise<FieldMap[]> {
+    await this.initialized; // This is ensuring that our initialization is complete
     const missing = this.missing(requested);
 
     if (missing.length || layout) {
       this.parameters.fields = missing.join(',');
       if (layout) {
         this.parameters.layout = layout;
       }
+      console.log('Making meta call for these fields', requested.toString(), this.endpoint);
```

Comment:
Do we need this console.log? I prefer not to have any console logs.

Reply:
yeah I'll take this out